misaligned.c

```diff
@@ -10,8 +10,8 @@ int printAndUpdateColorData() {
         for(j = 0; j < 5; j++) {
             localPairNumber = localPairNumber+1;
             strcpy(colorpair[localPairNumber].MajorColor , majorColor[i]);

```

Comment:
instead of separate `strcpy` and `printf`, you could use sprintf to format the string and and then simply print it to console. Then you can test the formatting / alignment as well.